demo/lit-html/index.html

```diff
@@ -2,6 +2,13 @@
 <html>
   <head>
     <meta charset="UTF-8">
+    <script type="importmap">
```

Comment:
I went ahead on modernized our demos to use import maps and bare-module specifiers for our dependencies.